akka-docs/src/main/paradox/remoting.md

```diff
@@ -408,8 +410,8 @@ finished.
 
 @@@ note
 
-In order to switch off the logging, set
-`akka.remote.log-remote-lifecycle-events = off` in your
+In order to switch of the logging, set
+`akka.remote.classic.log-remote-lifecycle-events = off` in your
```

Comment:
None of those debug logging toggles are available for artery?

Reply:
These ones are only used from Endpoint.scala and Remoting.scala which is classic. I can see we have tests that set this incorrectly so i'll fix those

Reply:
it's only classic

Reply:
Thx for verifying, I didn't know that before.